content/collective_intro.tex

```diff
@@ -55,7 +55,7 @@ \subsubsection*{Team-based collectives}
 \item \FUNC{shmem\_\{TYPE\_\}broadcast\{mem\}}
 \item \FUNC{shmem\_\{TYPE\_\}collect\{mem\}}
 \item \FUNC{shmem\_\{TYPE\_\}fcollect\{mem\}}
-\item Reduction routines for the following operations: AND, OR, XOR, MAX, MIN, SUM, PROD
+\item Reduction routines for the following operations: AND, MAX, MIN, SUM, PROD, OR, XOR
```

Comment:
Why the reordering? The new ordering isn't alphabetized, which would be: AND, MAX, MIN, OR, PROD, SUM, XOR. The old order was, as I saw it, ordered as {bitwise, comparison, arithmetic} operators.

Reply:
I reordered it to match with the description for active-set based collectives. It looks weird to me that they describe very similar contents but in different orders. Maybe it is better to use "AND, OR, XOR, MAX, MIN, SUM, PROD" for both.  Suggestion?

Reply:
Yes, we should reorder the line for the active-set based collectives to match that of the team-based collectives:

https://github.com/nspark/specification/blob/14e66bed2fceda1567310f9e2cb4b5bf15056250/content/collective_intro.tex#L115

This will match the ordering of the routine listings in the Specification itself.

Reply:
Fixed.